docs/developer-docs/web-apps/verifiable-credentials/overview.mdx

```diff
@@ -0,0 +1,47 @@
+---
+keywords: [intermediate, concept, identity, credentials, verifiable credentials]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Intermediate", "Concept"]} />
+
+A credential is a document or certificate that validates an individual's identity or qualifications. Credentials are a part of daily life, with common examples including:
+
+- A driver's license that qualifies an individual to operate a motor vehicle.
+- A university degree that can be used to assert a level of education.
+- A government-issued passport enabling travel between countries.
+
+These are examples of physical credentials, as they have a physical component that must be presented for validation. Credentials can also be digital, relying on authentication using certifications or digital signatures for validation, such as a username and password or ICP Internet Identity.
```

Comment:
There is something that doesn't sound totally right to me: "relying on authentication using certifications or digital signature for validation". A username and password is not a certification nor a digital signature. Nor is "ICP Internet Identity".

How about if we change it to:
> Credentials can also be digital, relying on authentication using cryptographic signatures such as canister signatures in Internet Computer.

Reply:
Revised to "Credentials can also be digital, relying on authentication using cryptographic signatures such as canister signatures on ICP."